billing_test.go

```diff
@@ -1,39 +1,37 @@
-package paypal_test
+package paypal
 
 import (
 	"fmt"
 	"time"
-
-	paypal "github.com/plutov/paypal"
 )
 
-func BillingExample() {
-	plan := paypal.BillingPlan{
+func billingExample() {
```

Comment:
Since it's an example, this function shouldn't be exposed in the `paypal` package, but should be visible only in godoc.